### PR TITLE
[FIX] Wrong positioning of popover when using RTL languages

### LINKED ISSUE #10068: Menu in RTL isnt in place
### Server Setup Information:

* Version of Rocket.Chat Server: 63
* Operating System: ubuntu
* Deployment Method(snap/docker/tar/etc): 
* Number of Running Instances: 
* DB Replicaset Oplog: 
* Node Version: 
* mongoDB Version: 

### Steps to Reproduce:

screenshot-open rocket chat-2018 03 09-04-53-00

### PR DESCRIPTION
**Pull request by ggazzo**
now you can just pass the anchor element as `currentTarget` and the script will find the right position

Closes #10068

before: 
image

after: 
image

### CODE REVIEW

File: packages/rocketchat-ui-sidenav/client/sidebarHeader.js [hunk 1]
```diff
@@ -105,13 +105,10 @@ const toolbarButtons = (user) => {
...
-				mousePosition: () => ({
-					x: e.currentTarget.getBoundingClientRect().left,
-					y: e.currentTarget.getBoundingClientRect().bottom + 50
-				}),
+				currentTarget: e.currentTarget,
 				customCSSProperties: () => ({
-					top:  `${ e.currentTarget.getBoundingClientRect().bottom + 10 }px`,
-					left: `${ e.currentTarget.getBoundingClientRect().left - 10 }px`
+					top:  `${ e.currentTarget.getBoundingClientRect().bottom + 10 }px`
+					// left: `${ e.currentTarget.getBoundingClientRect().left - 10 }px`
```
Comment (gdelavald) [inline packages/rocketchat-ui-sidenav/client/sidebarHeader.js]: Remove comments

File: packages/rocketchat-ui/client/views/app/popover.js [hunk 2]
```diff
@@ -75,7 +78,7 @@ Template.popover.onRendered(function() {
 			}
 
 			popoverContent.style.top = `${ top }px`;
-			popoverContent.style.left = `${ left }px`;
+			popoverContent.style.left = `${ isRtl() ? left : left }px`;
```
Comment (gdelavald) [inline packages/rocketchat-ui/client/views/app/popover.js]: Left and Left? No need for conditional then.

**Comment by ggazzo** [comment]
please remove the comments.

**Comment by fifou17** [comment]
still some issues in design 
screenshot-rocket-chat-pr-10428 herokuapp com-2018 04 12-07-01-02
screenshot-rocket-chat-pr-10428 herokuapp com-2018 04 12-07-01-49

**Comment by karlprieb** [comment]
The `top` property calc is wrong
image

**Pull request merged**